cmake/Modules/Documentation.cmake

```diff
@@ -3,9 +3,11 @@
 ###############################################################################
 option(BUILD_DOC "Build LAMMPS HTML documentation" OFF)
 if(BUILD_DOC)
-  find_package(PythonInterp 3 REQUIRED)
+  # Sphinx 3.0 requires at least Python 3.5
+  find_package(PythonInterp 3.5 REQUIRED)
```

Comment:
```suggestion
if(CMAKE_VERSION VERSION_LESS 3.12)
    find_package(PythonInterp 3.5 REQUIRED) # Deprecated since version 3.12
  else()
    find_package(Python REQUIRED COMPONENTS Interpreter)
  endif()
```